product_docs/docs/pgd/4/harp/09_consensus-layer.mdx

```diff
@@ -149,49 +149,47 @@ zones in regions over traditional data centers.
 
 The process of selecting a leader begins with the startup of the first HARP manager:
 
-* Starts Postgres on the node
-* Reads the Postgres status (including any data on replication lag)
-* Queries the current leader (there should be none)
-* Stores the leadership request in the DCS
-* If the leadership request in not rejected, store the leader lease in the DCS
-* Refresh the leader lease (on a configurable interval, default is 500ms)
+* Starts Postgres on the node.
+* Reads the Postgres status, including any data on replication lag.
+* Queries the current leader. (There isn't one.)
```

Comment:
```suggestion
* Queries the current leader. (There shouldn't be one for this to become the leader).
```

This is trying to point out how the leader will determine if it is the leader.

Reply:
The word "should" almost always creates ambiguity, and in this case I'm still unclear of the meaning. Is it saying this?

Queries the current leader. For this one to become the leader, there must not already be one. 

If so, I am wondering if we might add another sentence saying how to remedy it if there already is one?

Reply:
It's by design. There should not be a leader because if there was it would become a follower. 

```suggestion
* Queries for the current leader. No current leader must be returned for this node to become leader.
```